### ISSUE TITLE
LC recipes uncraftable while IE is installed.

### ISSUE THREAD

**Issue by Urijahuri** [SlimeKnights/TinkersConstruct#2643]
was playing on a modpack that incorperated TiC, Immersive Engineering, and Lightningcraft. THe issues comes with the fact that when you have IE installed it disables the recipes to smelt any of the Lightningcraft materials that were not already formed into tools.

both mod devs have been informed. informing there of the glitch as well.

**Comment by KnightMiner** [comment]
Can you rephrase the issue? I fail to see how missing Lightning craft recipes when Immersive Engineering is installed is a Tinkers Construct issue.

**Comment by bonii-xx** [comment]
Not TiC related, if I had to guess I'd say lightningcraft is doing something wrong, since I know that IE is using the proper APIs

**Comment by sblectric** [comment]
This was an error on my end, and I have fixed it in my latest release.
